### ISSUE TITLE
Example do not finish.

### ISSUE THREAD

**Issue by HeinOldewage** [chrislusf/gleam#59]
Running the CSV example in standalone mode on windows never finishes.
It is also not using any CPU. So it seem to be blocking somewhere.

Do you test on windows? What other information can I give?

**Comment by chrislusf** [comment]
Haven't tried it on Windows.

**Comment by lewgun** [comment]
yes, it's can't be running under windows, because, it uses Unix Shell to fork sub process

**Comment by DTherHtun** [comment]
How to connect hdfs ?

**Comment by devsohi** [comment]
Example is not even finished on linux . I am using ubuntu 15 . I see no output on terminal when running wordcount in local mode
